### PR TITLE
Prepare 0.14.0 release

### PR DESCRIPTION
**Pull request by postrational**

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -33,25 +33,36 @@ You can follow these instructions to build an nGraph Python wheel containing bot
...
+Clone nGraph's `v0.13.0` tag, build and install it into `$HOME/ngraph_dist`:
 
-    $ git clone -b 'v0.10.0' --single-branch --depth 1 https://github.com/NervanaSystems/ngraph.git
+    $ cd # Change directory to where you would like to clone nGraph sources
+    $ git clone -b 'v0.13.0' --single-branch --depth 1 https://github.com/NervanaSystems/ngraph.git
     $ mkdir ngraph/build
     $ cd ngraph/build
-    $ cmake ../ -DNGRAPH_USE_PREBUILT_LLVM=TRUE -DCMAKE_INSTALL_PREFIX=$HOME/ngraph_dist -DNGRAPH_ONNX_IMPORT_ENABLE=TRUE
+    $ cmake ../ -DCMAKE_INSTALL_PREFIX=$HOME/ngraph_dist -DNGRAPH_ONNX_IMPORT_ENABLE=TRUE -DNGRAPH_USE_PREBUILT_LLVM=TRUE 
     $ make
```
Comment (arogowie-intel) [inline BUILDING.md]: Maybe we should limit here used cores? ie:
```
`lscpu --parse=CORE | grep -v '#' | sort | uniq | wc -l`
```

File: BUILDING.md [hunk 2]
```diff
@@ -33,25 +33,36 @@ You can follow these instructions to build an nGraph Python wheel containing bot
...
+    (nGraph) $ 
+
 Build Python package (Binary wheel) for nGraph:
 
-    $ cd ngraph/python
-    $ git clone --recursive -b allow-nonconstructible-holders https://github.com/jagerman/pybind11.git
-    $ export PYBIND_HEADERS_PATH=$PWD/pybind11
-    $ export NGRAPH_CPP_BUILD_PATH=$HOME/ngraph_dist
-    $ python3 setup.py bdist_wheel
+    (nGraph) $ cd # Change directory to where you would like to clone nGraph sources
```
Comment (arogowie-intel) [inline BUILDING.md]: Probably you meant: "Change directory to where you *have cloned* nGraph sources"

File: BUILDING.md [hunk 3]
```diff
@@ -61,7 +72,7 @@ Once the Python binary wheel file (`ngraph-*.whl`) is prepared you can install i
 
 For example:
 
-    (your_venv) $ pip install -U dist/ngraph-0.10.0-cp35-cp35m-linux_x86_64.whl
+    (nGraph) $ pip install -U dist/ngraph_core-0.0.0.dev0-cp35-cp35m-linux_x86_64.whl
```
Comment (arogowie-intel) [inline BUILDING.md]: If we build wheel won't we have a specific package version in wheel name?

File: BUILDING.md (hunk 2 above)
Comment (postrational) [reply BUILDING.md]: Copy-paste error :)

File: BUILDING.md (hunk 3 above)
Comment (postrational) [reply BUILDING.md]: Not if we use `--single-branch --depth 1 `

File: BUILDING.md (hunk 1 above)
Comment (postrational) [reply BUILDING.md]: No, I think we should recommend single-process compilation. A user can increase this at their own risk.

**Pull request closed without merging**